cypress/integration/main_page_spec.js

```diff
@@ -0,0 +1,24 @@
+
+describe ('Main Page', () => {
+
+    beforeEach(() => {
+        cy.visit('http://localhost:3000');
+      });
+
+    describe('Main Page Load', () => {
+        
+        it('should be able to visit the app and display the header', () => {
+            cy.get('h1')
+            .contains('Rancid Tomatillos')
+        })
+
+        it('should be able to visit the app and render all movies', () => {
+            cy.fixture('/all_movie_data.json').then((data) => {
```

Comment:
This is an interesting addition, with the 'cy.fixture.' It looks like this code isn't working quite the way you thought it would. We could probably trying playing with this json file and maybe try a .js file instead and see if that changes anything. Either that or perhaps we try to target a different, more specific element.